README.md

```
@@ -1,16 +1,115 @@
+```shell
+mvn install -DskipTests -Dpowsybl.docker.install
 ```
-mvn package -DskipTests && rm -f src/main/resources/geoData.sql && java  -jar target/gridsuite-geo-data-server-1.0.0-SNAPSHOT-exec.jar --spring.jpa.properties.jakarta.persistence.schema-generation.scripts.action=create 
+
+Please read [liquibase usage](https://github.com/powsybl/powsybl-parent/#liquibase-usage) for instructions to automatically generate changesets. After you generated a changeset do not forget to add it to git and in `src/resources/db/changelog/db.changelog-master.yml`.
+
+---
+
+## Interactions with Other Microservices
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)